nototools/mti_cmap_data.py

```diff
@@ -81,24 +81,92 @@ def cmap_data_from_csv_file(csvfile):
   return cmap_data_from_csv(csvdata, csvfile)
 
 
+def csv_to_xml(csv_file, xml_file, scripts):
+  cmapdata = cmap_data_from_csv_file(csv_file)
+  if scripts:
+    print >> sys.stderr, 'not filtering by scripts when converting csv to xml'
```

Comment:
It was a little confusing to me what this message was for at first. I might make it more explicit, like "not using 'scripts' argument when..."